package.json

```diff
@@ -21,6 +22,7 @@
   "license": "AGPL-3.0",
   "dependencies": {
     "@solidjs/router": "^0.6.0",
+    "localforage": "^1.10.0",
```

Comment:
Committed by accident? This package doesn't seen to be used anywhere.

Reply:
accident, my bad.